mangaki/mangaki/views.py

```diff
@@ -807,6 +809,78 @@ def legal_mentions(request):
     return render(request, 'mangaki/legal.html')
 
 
+def fix_index(request):
+    suggestion_list = Suggestion.objects.select_related('work', 'user').all().prefetch_related(
```

Comment:
I'm not sure but I think that you could put `prefetch_related` and `order_by` before the `all` call.